src/adaptors/bagful/index.js

```diff
@@ -0,0 +1,130 @@
+const { default: BigNumber } = require('bignumber.js');
+const utils = require('../utils');
+const sdk = require('@defillama/sdk');
+
+let tokenInfoMapping = {
+  0: {
+    symbol: 'MENDI',
+    decimals: 18,
+    isFixedRate: false,
+    rewardTokens: ['0xA219439258ca9da29E9Cc4cE5596924745e12B93'],
+    dailyRewardAmount: 18,
```

Comment:
woulnd't this be variable given its not a fixed rate? 
why not just fetch from the contracts instead of hardcoding